lib/ui/window.dart

```diff
@@ -54,15 +54,24 @@ part of dart.ui;
 ///
 /// See also:
 ///
-///  * [FlutterWindow], a special case of a [FlutterView] that is represented on
+///  * [FlutterView], a special case of a [FlutterView] that is represented on
```

Comment:
A FlutterView is a special case of a FlutterView? :P

Reply:
😅